charts/pega/config/deploy/prconfig.xml

```diff
@@ -7,6 +7,7 @@
         <env name="database/databases/PegaDATA/dataSource" value="java:comp/env/jdbc/PegaRULES"/>
         <env name="security/urlaccesslog" value="NORMAL" />
         <env name="security/urlaccessmode" value="WARN" />
+        <env name="http/reversednslookup" value="false" />​
```

Comment:
@raghavf1 Is it mandatory to set reverse DNS lookup to` false `by default? or can this be configurable from values.yaml? 
As per the user story and bug item mentioned in #240, it is under Engine backlog. 

Can you confirm if this was verified with Engine teams? Some approvals from Engine can help us proceeding with merge.